server/game/cards/02-AoA/WayOfThePorcupine.js

```diff
@@ -0,0 +1,13 @@
+const Card = require('../../Card.js');
+
+class WayOfThePorcupine extends Card {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.addKeyword({ hazardous: 3 })
+        });
+    }
+}
+
+WayOfThePorcupine.id = 'way-of-the-porcupine';
+
+module.exports = WayOfThePorcupine;
```

Comment:
The linting rules require that there's a new line at the end of the file - other than that, it looks fine

If you have an editor that supports eslint, it should be able to auto fix most issues on save